bundles/org.openhab.binding.velbus/src/main/java/org/openhab/binding/velbus/internal/handler/VelbusVMBDALIHandler.java

```diff
@@ -0,0 +1,399 @@
+ * sent to one of the channels.
+ *
+ * @author Daniel Rosengarten - Initial contribution
+ */
+@NonNullByDefault
+public class VelbusVMBDALIHandler extends VelbusSensorWithAlarmClockHandler {
+    public static final Set<ThingTypeUID> SUPPORTED_THING_TYPES = new HashSet<>(
+            Arrays.asList(THING_TYPE_VMBDALI, THING_TYPE_VMBDALI_20));
+    private @Nullable ScheduledFuture<?> refreshJob;
+    private @NonNullByDefault({}) VelbusSensorConfig sensorConfig;
+
+    private VelbusColorChannel[] colorChannels;
+    private VelbusVirtualColorChannel[] virtualColorChannels;
+
+    public VelbusVMBDALIHandler(Thing thing) {
+        super(thing, 9);
+
+        colorChannels = new VelbusColorChannel[81];
+        virtualColorChannels = new VelbusVirtualColorChannel[16];
+    }
+
+    @Override
+    public void initialize() {
+        this.sensorConfig = getConfigAs(VelbusSensorConfig.class);
+
+        super.initialize();
+
+        initializeAutomaticRefresh();
+        initializeColorChannel();
+        initializeVirtualLight();
+        initializeChannelStates();
+    }
+
+    private void initializeAutomaticRefresh() {
+        int refreshInterval = this.sensorConfig.refresh;
+
+        if (refreshInterval > 0) {
+            startAutomaticRefresh(refreshInterval);
+        }
+    }
+
+    private void initializeColorChannel() {
+        for (int i = 0; i <= 80; i++) {
+            colorChannels[i] = new VelbusColorChannel();
+        }
+    }
+
+    private void initializeVirtualLight() {
+        String virtualLight;
+
+        for (int i = 1; i <= 16; i++) {
+            if (getConfig().containsKey(VIRTUAL_LIGHT + i)) {
+                virtualLight = getConfig().get(VIRTUAL_LIGHT + i).toString();
+                if (virtualLight.length() > 0) {
+                    try {
+                        virtualColorChannels[i - 1] = new VelbusVirtualColorChannel(virtualLight);
+                    } catch (Exception e) {
+                        logger.error("VMBDALI on address {} : Virtual Light {} has wrong channel format '{}'. {}",
+                                getModuleAddress().getAddress(), i, virtualLight, e.getMessage());
+                    }
+                }
+            }
+            if (virtualColorChannels[i - 1] == null) {
+                virtualColorChannels[i - 1] = new VelbusVirtualColorChannel();
+            }
+        }
+    }
+
+    private void initializeChannelStates() {
+        VelbusBridgeHandler velbusBridgeHandler = getVelbusBridgeHandler();
+        if (velbusBridgeHandler == null) {
+            updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.BRIDGE_OFFLINE);
+            return;
+        }
+
+        sendDaliReadoutRequest(velbusBridgeHandler, ALL_DALI_CHANNELS);
+    }
+
+    @Override
+    public void dispose() {
+        final ScheduledFuture<?> refreshJob = this.refreshJob;
+        if (refreshJob != null) {
+            refreshJob.cancel(true);
+        }
```

Comment:
```suggestion
        this.refreshJob = null;
        }
```

Reply:
Was marked as resolved, but i don't see code change or comment.

Reply:
I've accept your code suggestion, but it introduced a "code style error". 
I've corrected this in the commit 5fe388c.